hack/modify-test-images.sh

```diff
@@ -7,3 +7,5 @@ ${SED_BIN} -i "s#local/opentelemetry-operator-targetallocator:e2e#${TARGETALLOCA
 ${SED_BIN} -i "s#local/opentelemetry-operator-targetallocator:e2e#${TARGETALLOCATOR_IMG}#g" tests/e2e/prometheus-config-validation/*.yaml
 
 ${SED_BIN} -i "s#local/opentelemetry-operator:e2e#${OPERATOR_IMG}#g" tests/e2e-multi-instrumentation/*.yaml
+
+${SED_BIN} -i "s#local/opentelemetry-operator-opamp-bridge:e2e#${OPERATOROPAMPBRIDGE_IMG}#g" tests/e2e/opampbridge/*.yaml
```

Comment:
you'll need to modify `prepare-e2e` in the makefile to pass this image in 

https://github.com/open-telemetry/opentelemetry-operator/blob/main/Makefile#L221-L223